### PR TITLE
update theme and branding

### PR DESCRIPTION
**Pull request by carlosceia**

### CODE REVIEW

File: themes/c8ydocs/layouts/shortcodes/c8y-current-version.html [hunk 1]
```diff
@@ -1 +1 @@
-{{- "2024.0" -}}
```
Comment (BeateRixen) [inline themes/c8ydocs/layouts/shortcodes/c8y-current-version.html]: This must stay 2024.0

File: themes/c8ydocs/layouts/shortcodes/link-apama-webhelp.html [hunk 2]
```diff
@@ -1 +1 @@
-{{- "https://documentation.softwareag.com/pam/10.15.4/en/webhelp/pam-webhelp/" -}}
+{{- "https://documentation.softwareag.com/pam/10.15.5/en/webhelp/pam-webhelp/" -}}
```
Comment (BeateRixen) [inline themes/c8ydocs/layouts/shortcodes/link-apama-webhelp.html]: This must stay as is

File: themes/c8ydocs/layouts/shortcodes/link-apamadoc-api.html [hunk 3]
```diff
@@ -1 +1 @@
-{{- "https://documentation.softwareag.com/pam/10.15.4/en/webhelp/related/ApamaDoc/" -}}
+{{- "https://documentation.softwareag.com/pam/10.15.5/en/webhelp/related/ApamaDoc/" -}}
```
Comment (BeateRixen) [inline themes/c8ydocs/layouts/shortcodes/link-apamadoc-api.html]: This must stay as is

File: themes/c8ydocs/layouts/shortcodes/c8y-current-version.html (hunk 1 above)
Comment (carlosceia) [reply themes/c8ydocs/layouts/shortcodes/c8y-current-version.html]: changed

File: themes/c8ydocs/layouts/shortcodes/link-apama-webhelp.html (hunk 2 above)
Comment (carlosceia) [reply themes/c8ydocs/layouts/shortcodes/link-apama-webhelp.html]: changed

File: themes/c8ydocs/layouts/shortcodes/link-apamadoc-api.html (hunk 3 above)
Comment (carlosceia) [reply themes/c8ydocs/layouts/shortcodes/link-apamadoc-api.html]: changed

**Pull request merged**